athena/assessment_module_manager/assessment_module_manager/med_api/models.py

```
@@ -0,0 +1,175 @@
+from __future__ import annotations
+
+from datetime import datetime
+from enum import Enum
+from typing import Any
+
+from pydantic import AnyUrl, BaseModel, ConfigDict, Field
```

Flake8 (7.3.0)

[error] 'pydantic.Field' imported but unused

(F401)